source/features/filter-pr-by-build-status.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {checkInline} from '../libs/icons';
+import features from '../libs/features';
+
+function init() {
+	const statusFilter = select('.table-list-header-toggle > details:nth-last-child(3)')!.cloneNode(true) as HTMLDetailsElement;
```

Comment:
```suggestion
	const statusFilter = select<HTMLDetailsElement>('.table-list-header-toggle > details:nth-last-child(3)')!.cloneNode(true);
```

Reply:
I have added `HTMLDetailsElement` in the generic but I'm not removing the `as HTMLDetailsElement`. `select` is complaining
```
Argument of type 'Node' is not assignable to parameter of type 'Element | DocumentFragment | Document | Window | undefined'.
```
I think we need to fix types for `select-dom` and allow `Node` as a possible type.

Reply:
It's actually just "yet another TS quirk": https://github.com/Microsoft/TypeScript/issues/283 ...

Reply:
But yes, `select` should support a `Node` type too. // @bfred-it

Reply:
Node doesn’t have querySelector, it would be like supporting `createTextNode().querySelector()` which would fail